doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -472,3 +453,178 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
+
+   * - ``hostbridge``
+     - Virtualized PCI hostbridge, a hardware bridge between the CPU's
+       high-speed system local bus and the Peripheral Component Interconnect
+       (PCI) bus.
+
+   * - ``virtio-blk``
+     - Virtio block type device, a string could be appended to specify the disk
+       image, e.g., ``virtio-blk,/root/test.img``.  You can also could use
+       ``nodisk`` to create a virtio-blk device with a dummy backend.
+
+   * - ``virtio-coreu``
+     - Used for Protected Audio Visual Path (PAVP) session management to provide
+       a User VM with Protected Audio Visual Path service.
+
+   * - ``virtio-input``
+     - Virtio type device to emulate input device. ``evdev`` char device node
+       should be appended, e.g., ``-s
+       n,virtio-input,/dev/input/eventX[,serial]``. ``serial`` is an optional
+       string used as the Uniq of guest virtio input device.
+
+   * - ``virtio-ipu``
+     - Virtio image processing unit (IPU) type device, such as a camera.
+
+   * - ``virtio-console``
+     - Virtio console type device for data input and output.
+
+   * - ``virtio-hyper_dmabuf``
+     - Virtio device that allows sharing data buffers between VMs using a
+       dmabuf-like interface.
+
+   * - ``virtio-hdcp``
+     - Virtio High-bandwidth Digital Content Protection (HDCP) type device. HDCP
+       is technology intended to protect unauthorized duplication of high
+       definition (HD) video and audio as it travels across connections.
+
+   * - ``virtio-heci``
+     - Virtio Host Embedded Controller Interface, parameters should be appended
+       with the format ``<bus>:<device>:<function>,d<0~8>``.
+
+   * - ``virtio-i2c``
+     - Virtio i2c type device,parameters with format:
+       ``<bus>[:<client_addr>[@<node>]][,<bus>[:<client_addr>[@<node>]]``
+
+       * ``<bus>`` specifies the bus number for the native I2C adapter, e.g.,
+         ``2`` means ``/dev/i2c-2``.
+       * ``<client_addr>`` specifies the address for the native client devices
+         such as ``1C`` or  ``2F``.
+       * ``@`` specifies the prefix for the ACPI node.
+       * ``<node>`` specifies the ACPI node name supported in the
+         ``acpi_node_table[]`` in the source code: only ``cam1``, ``cam2``, and
+         ``hdac`` are supported for MRB and  are platform-specific.
+
+   * - ``virtio-gpio``
+     - Virtio GPIO type device, parameters format is:
+       ``virtio-gpio,<@controller_name{offset|name[=mapping_name]:offset|name[=mapping_name]:...}@controller_name{...}...]>``
+
+       * ``controller_name``: use the command ``ls /sys/bus/gpio/devices`` to
+         check the native GPIO controller information.  Usually, the devices
+         represent the ``controller_name`` that you can use. You can also use
+         the command ``cat /sys/bus/gpio/device/XXX/dev`` to get the device id
+         that can be used to match ``/dev/XXX``, and then use ``XXX`` as the
+         ``controller_name``. On MRB and Intel NUC platforms,
```

Comment:
Same comment here, We need a platform name that's more specific than ``MRB``, and is this true for all NUCs?

Reply:
APL platform updated in doc